src/components/Card/Card.jsx

```diff
@@ -3,29 +3,62 @@ import styled from "styled-components";
 
 import data from "../../data.json";
 
-const CardStyled = styled.img`
-  width: 200px;
-  height: 200px;
-  border-radius: 10px;
-  box-shadow: 5px 5px 10px black;
-  cursor: pointer;
-  @media (max-width: 800px) {
-    width: 120px;
-    height: 120px;
+const CardStyled = styled.div`
+  perspective: 1000px;
+  padding-top: 0;
+
+  .card {
+    position: relative;
+    cursor: pointer;
+    transition: 0.6s;
+    transform-style: preserve-3d;
+    position: relative;
+
+    .card-image {
+      width: 175px;
+      height: 175px;
+      border-radius: 10px;
+      box-shadow: 5px 5px 10px black;
+      cursor: pointer;
+
+      @media (max-width: 800px) {
+        width: 120px;
+        height: 120px;
+      }
+
+      @media (max-width: 500px) {
+        width: 75px;
+        height: 75px;
+      }
+    }
   }
-  @media (max-width: 500px) {
-    width: 75px;
-    height: 75px;
+
+  .front {
+    transform: rotateY(0deg);
+  }
+
+  .back {
+    transform: rotateY(180deg);
+  }
+
+  .card.is-flipped {
+    transform: rotateY(180deg);
   }
 `;
 
 function Card({ image, cardId, isFlipped, handleClick }) {
   return (
-    <CardStyled
-      onClick={handleClick}
-      src={isFlipped ? image : data.cover.src}
-      data-id={cardId}
-    />
+    <CardStyled>
+      <div className={isFlipped ? "is-flipped card" : "card"}>
```

Comment:
```js
<div className={isFlipped ? "is-flipped card" : "card"}>
```
Kod tekrarı yapmamak için burayı şöyle yapabilirsin.

```js
<div className={`card ${isFlipped ? "is-flipped" : ""}`}>
```

ya da bunları daha düzenli yazmanı sağlayan [classnames](https://www.npmjs.com/package/classnames) gibi bir paketi kullanabilirsin.

Eğer onu kullansaydın şöyle olacaktı.

```js
import cx from "classnames";

<div className={cx("card", { "is-flapped": isFlapped })}>
```